diff --git a/CMakeLists.txt b/CMakeLists.txt
index c9ad81c..458c0ee 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -123,9 +123,9 @@ set_source_files_properties(${PROJECT_BINARY_DIR}/doc/edge.8.gz
                             PROPERTIES GENERATED 1)
 
 install(FILES ${PROJECT_BINARY_DIR}/doc/edge.8.gz
-        DESTINATION /usr/share/man8)
+        DESTINATION /usr/share/man/man8)
 install(FILES ${PROJECT_BINARY_DIR}/doc/supernode.1.gz
-        DESTINATION /usr/share/man1)
+        DESTINATION /usr/share/man/man1)
 install(FILES ${PROJECT_BINARY_DIR}/doc/n2n_v2.7.gz
-        DESTINATION /usr/share/man7)
+        DESTINATION /usr/share/man/man7)
 endif(DEFINED UNIX)
