diff --git a/dlls/d3d9/surface.c b/dlls/d3d9/surface.c
index eb78a01..f14ec5e 100644
--- a/dlls/d3d9/surface.c
+++ b/dlls/d3d9/surface.c
@@ -91,8 +91,16 @@ static ULONG WINAPI IDirect3DSurface9Impl_Release(IDirect3DSurface9 *iface)
         TRACE("(%p) : Forwarding to %p\n", This, This->forwardReference);
         return IUnknown_Release(This->forwardReference);
     } else {
+        ULONG ref;
+
+        if (This->ref == 0)
+        {
+            WARN("Surface doesn't have any references.\n");
+            return 0;
+        }
+
         /* No container, handle our own refcounting */
-        ULONG ref = InterlockedDecrement(&This->ref);
+        ref = InterlockedDecrement(&This->ref);
 
         TRACE("%p decreasing refcount to %u.\n", iface, ref);
 
